tests/installation/setup_online_repos.pm

```diff
@@ -20,8 +20,13 @@ use version_utils;
 
 sub run {
     # ordered according to real repos lists
-    my @default_repos = qw(update-non-oss update-oss main-non-oss main-oss debug-main untested-update debug-update source);
-
+    my @default_repos = qw();
+    if (is_leap('<15.1')) {
+        @default_repos = qw(update-non-oss update-oss main-non-oss main-oss debug-main untested-update debug-update source);
```

Comment:
can be:
```if () {
my @d......}
else {
my @d....}
````

Reply:
ahh, my bad habitat from another langs :D

Reply:
Your suggestion would fail with `Global symbol "@default_repos" requires explicit package name (did you forget to declare "my @default_repos"?)`.